principles/fp-009-users.md

```diff
@@ -38,42 +62,56 @@ You may have multiple examples for each user, and mulitple users under the `usag
 
 ## Examples
 
-- Use of the target ontology's term IRIs in other ontologies. This can
-  be evidenced by linking to the other ontology that uses an ontology
-  term IRI from this ontology
-
-- Imports in other ontologies, again, evidenced through a link to the
-  importing ontology
-
-- A documentation page with links to databases using the ontology for
-  annotation
-
-- Use in semantic web projects (e.g., Open PHACTS usage)
-
-- Use in diverse software applications, including text mining or
-  analysis pipelines
-
-- Publications showing the ontology being used in research (evidenced
-  by citation details for the publications; if behind paywall it may
-  be necessary to provide a PDF for the paper)
-
-- Citations to the ontology publication(s); such citations are only
-  relevant if the authors indicate actual use of the cited ontology
-  within some community (mere mention of the existence is not enough
-  to warrant inclusion)
-
-- Documentation of requests from multiple users, e.g., via an issue
-  tracker (provide a link to the issue tracker online)
-
-- Use of the terms from the ontology to structure or annotate
-  experimental or derived data (e.g. GOA; provide details of how to
-  review the annotations)
+Use of PSI-MOD term IRI (in the form of a CURIE) in the PR ontology:
+```
```

Comment:
```suggestion
```yaml
```

would be great to use code highlighting in these examples :)

Reply:
@cthoyt I'm afraid I don't know what you mean. Is it just to add 'yaml' ? What does that do?

Reply:
yes, if you follow the triple ticks with the language name or data format name, the content will get beautifully highlighted :)

Reply:
Done!